w13h03/minijvm/mod.jvm

```diff
@@ -0,0 +1,10 @@
+//#D should modulo correctly
+//#! R15 R7 W1
+//#! R30 R5 W0
+//#! R100 R1 W0
```

Comment:
```suggestion
//#! R7 R15 W1
//#! R5 R30 W0
//#! R1 R100 W0
```

Reply:
That's weird though 🤔

Reply:
> That's weird though 🤔 

Yeah, Robin himself said that it wasnt meant to be like that and that the operands were accidentally swapped around iirc